docker-build-image

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Copyright 2020 Northern.tech AS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -e
+
+IMAGE_NAME_PREFIX=mendersoftware/mender-dist-packages:builder
+
+# Parse args
+if [ $# -ne 3 ]; then
+    echo "usage: $0 distro release arch"
+    exit 1
+fi
+DISTRO="${1}"
+RELEASE="${2}"
+ARCH="${3}"
+
+# Pass GPG key as build argument if set
+if [ -n "${GPG_KEY_BUILD}" ]; then
+    docker build \
+            --tag ${IMAGE_NAME_PREFIX}-${DISTRO}-${RELEASE}-${ARCH} \
+            --build-arg DISTRO=${DISTRO} \
+            --build-arg VERSION=${RELEASE} \
+            --build-arg ARCH=${ARCH} \
+            --build-arg GPG_KEY_BUILD="${GPG_KEY_BUILD}" \
```

Comment:
Do as you wish with suggestion:
Can do `{GPG_KEY_BUILD:+--build-arg GPG_KEY_BUILD="${GPG_KEY_BUILD}"}`

Reply:
Thanks, I will do this, much cleaner! :+1: